homeassistant/__main__.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import threading
 
-from typing import Optional, List, Dict, Any  # noqa #pylint: disable=unused-import
```

Comment:
Is this needed?

From PyLint changelog:

> Don't emit unused-import anymore for typing imports used in type comments.

Reply:
Yes, it's still needed. Note _used in type comments_ in the pylint changelog -- in this case, `Optional`, `Dict`, and `Any` are not used anywhere. (In my opinion, these cases should be fixed for real by just dropping the actually unused imports, but that's slightly off topic for this particular PR.)

Reply:
I've pushed a "Revert pylint 2 inline disable syntax fixes addressing unused-imports" commit to the source branch, but it seems github is having issues at the moment and that commit doesn't show up here yet. Not sure if I should do something about it, but will wait at least a bit first.